### ISSUE TITLE
Unable to discover / create steps

### ISSUE THREAD

**Issue by snow-jallen** [specsolutions/deveroom-visualstudio#38]
After adding a new feature file, and writing the text of the given/when/then statements, I right clicked on one of the statements to generate the step file.  The following error is generated:
```
Command executed:
  C:\Users\jonathan\source\repos\AnnealingValidator\AnnealeatingValidator.Tests\bin\Debug\netcoreapp3.0> C:\Program Files\dotnet\dotnet.exe exec C:\USERS\JONATHAN\APPDATA\LOCAL\MICROSOFT\VISUALSTUDIO\16.0_FC286A5D\EXTENSIONS\ZI44EQ4M.Y42\Connectors\V2\deveroom-specflow-v2.dll discovery C:\Users\jonathan\source\repos\AnnealingValidator\AnnealeatingValidator.Tests\bin\Debug\netcoreapp3.0\AnnealingValidator.Tests.dll ""
Exit code: 4
Message: 
Error: Exception has been thrown by the target of an invocation. -> Method not found: 'BoDi.IObjectContainer TechTalk.SpecFlow.Infrastructure.ContainerBuilder.CreateGlobalContainer(TechTalk.SpecFlow.Configuration.IRuntimeConfigurationProvider)'.
Exception: System.Reflection.TargetInvocationException->System.MissingMethodException
StackTrace:
   at Deveroom.VisualStudio.SpecFlowConnector.Discovery.V3000.SpecFlowV3000Discoverer.GetBindingRegistry(Assembly testAssembly, String configFilePath)
   at Deveroom.VisualStudio.SpecFlowConnector.Discovery.BaseDiscoverer.DiscoverInternal(Assembly testAssembly, String testAssemblyPath, String configFilePath) in W:\Deveroom\Deveroom.VisualStudio\Deveroom.VisualStudio.SpecFlowConnector\Discovery\BaseDiscoverer.cs:line 29
   at Deveroom.VisualStudio.SpecFlowConnector.Discovery.BaseDiscoverer.Discover(Assembly testAssembly, String testAssemblyPath, String configFilePath) in W:\Deveroom\Deveroom.VisualStudio\Deveroom.VisualStudio.SpecFlowConnector\Discovery\BaseDiscoverer.cs:line 21
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Deveroom.VisualStudio.SpecFlowConnector.ReflectionExtensions.ReflectionCallMethod[T](Object obj, String methodName, Type[] parameterTypes, Object[] args) in W:\Deveroom\Deveroom.VisualStudio\Deveroom.VisualStudio.SpecFlowConnector\ReflectionExtensions.cs:line 18
   at Deveroom.VisualStudio.SpecFlowConnector.Discovery.ReflectionSpecFlowDiscoverer.Discover(Assembly testAssembly, String testAssemblyPath, String configFilePath) in W:\Deveroom\Deveroom.VisualStudio\Deveroom.VisualStudio.SpecFlowConnector.V2\Discovery\ReflectionSpecFlowDiscoverer.cs:line 25
   at Deveroom.VisualStudio.SpecFlowConnector.Discovery.DiscoveryProcessor.Process() in W:\Deveroom\Deveroom.VisualStudio\Deveroom.VisualStudio.SpecFlowConnector.V2\Discovery\DiscoveryProcessor.cs:line 41
   at Deveroom.VisualStudio.SpecFlowConnector.ConsoleRunner.EntryPoint(String[] args) in W:\Deveroom\Deveroom.VisualStudio\Deveroom.VisualStudio.SpecFlowConnector\ConsoleRunner.cs:line 32

Warning: InvokeDiscovery: The project bindings (e.g. step definitions) could not be discovered. Navigation, step completion and other features are disabled. 
  Please check the error message above and report to https://github.com/specsolutions/deveroom-visualstudio/issues if you cannot fix.
Warning: ShowProblem: User Notification: All steps have been defined in this file already.
```

**Comment by Jordank321** [comment]
I had the same issue yesterday.
I managed to resolve this by downgrading my specflow pacakges to version `.161-beta`

**Comment by gasparnagy** [comment]
Thx. I will try to repro & fix it this evening.

**Comment by gasparnagy** [comment]
I can confirm that the issue is reproducible with SpecFlow v3.0.220. Restoring to SpecFlow 3.0.213 or earlier solves the problem.
I am working on the fix.

**Comment by gasparnagy** [comment]
Fixed. I hope to release it tonight.

**Comment by gasparnagy** [comment]
The fix has just been released as v1.3.0.
